arch/x86/mmu.c

```diff
@@ -265,7 +252,7 @@ static uint32_t map_mem_region(void *vaddr, void *paddr,
 		}
 		case PAGING_REQUEST_TYPE_UNMAP:
 		{
-			if (tmp & table_present) {
+			if (entry) {
```

Comment:
The check is only for PAGING_REQUEST_TYPE_UNMAP.
Can we always clear the entry to zero for like what is done for PAGING_REQUEST_TYPE_MAP?
In such case the MEM_READ64 can be reduced.

Reply:
I will add 'invalidate TLB' in these 3 cases in another patch, then it is necessary to check the entry here